content/blog/monthly-december-2019.md

```diff
@@ -0,0 +1,93 @@
+---
+title: This Month in TiKV - December 2019
+date: 2020-1-7
+author: Ana Hobden
+---
+
+<!-- Fill in the below from the API: https://developer.github.com/v4/explorer/ -->
+
+Happy new year and happy new decade! How did our contributors wrap up the decade? Let's take a look, and then take a peek at what's coming this year!
+
+## Releases
+
+This month our team made 4 TiKV releases - 3.0.x and 2.1.x releases are minor releases, and 3.1.0-beta is our upcoming next major release before 4.0.
+
+You can review the changelogs here:
+
+* [3.0.7](https://github.com/tikv/tikv/releases/tag/v3.0.7)
+* [3.0.8](https://github.com/tikv/tikv/releases/tag/v3.0.8)
+* [2.1.19](https://github.com/tikv/tikv/releases/tag/v2.1.19)
+* [3.1.0-beta](https://github.com/tikv/tikv/releases/tag/v3.1.0-beta)
+
+Upgrading? Here's things you should know:
+
+* In **3.0.8** we reduced the severity of errors occuring in the coprocessor to `warn`.
+* In **3.0.8** the default value of `split-region-on-table` is now `false` to help reduce region creation pressure during table creation/deletion.
+* In **3.0.8** we now welcome you to experiment with a new feature to get flame graphs from a running TiKV. Try visiting `$SERVER:20180/debug/pprof/profile?seconds=10` to see a flamegraph of that TiKV. We'll be adding more details to our documentation about this soon!
+* Version **3.1.0** is a beta and should not be adopted in production deployments. Please do try it out and let us know if you find issues. It includes new follower read and backup/restore features.
+
+## News
+
+TiKV 3.1.0 is now in beta, with a stable release coming soon! This includes new major features like follower read and distributed backup/restore.
+
+<<<<<<< HEAD
```

Comment:
There are some conflicting texts here. Otherwise, it looks good to me.

Reply:
Whoops